includes/menu.inc

```diff
@@ -76,16 +76,9 @@ function oe_bootstrap_theme_preprocess_links__dropbutton(&$variables) {
   // Get the first link and use it for the dropbutton.
   $link = reset($links);
 
-  /** @var \Drupal\Core\Url $url */
   $variables['split'] = FALSE;
-  if (isset($link['link']) && ($url = $link['link']['#url'])) {
-    $button = $link['link'];
-
-    if ($variables['split'] = $url->getRouteName() !== '<nolink>') {
-      $button['#options']['attributes']['class'][] = 'btn';
-      $button['#options']['attributes']['class'][] = 'btn-sm';
-      $button['#options']['attributes']['class'][] = 'btn-outline-primary';
-    }
+  if (isset($link['text'])) {
```

Comment:
what are these changes dropbutton and dowpdown changes for ? seems out of scope of this ticket (if these are needed do report in a relevant ticket

Reply:
This is needed for testing. With oe_bootstrap_theme enabled with this code when you are in a create node page  the list of available paragraphs is empty.

Reply:
Create a new ticket: https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-961